tofu/__init__.py

```diff
@@ -54,12 +54,6 @@
 import warnings
 from .version import __version__
 
-# For tests without display with nosetests
-if 'matplotlib.pyplot' not in sys.modules.keys():
```

Comment:
Removed to avoid that tofu messes up with the backend, right ?

Reply:
yes